frontend/demo/src/actions/user/index.ts

```diff
@@ -30,12 +30,13 @@ export function loginViaEmail(requestPayload: LoginViaEmailRequestPayload) {
   return async (dispatch: Dispatch<any>) => {
     return doFetchFromBackend('users.login', requestPayload)
       .then((response: UserPayload) => {
-        dispatch(setCurrentUserAction(userMapper(response)));
-        return true;
+        const user = userMapper(response);
+        dispatch(setCurrentUserAction(user));
+        return Promise.resolve(user);
```

Comment:
I am not sure what the generated code will return... @chrismatix maybe you can write here what is wrong?

Reply:
Sorry, I don't understand the question. For now we only generate typings for the message content model.

Reply:
so its up to us to generate the code that talks to the api? then i will keep it this way.